marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/JobsController.java

```diff
@@ -62,12 +65,38 @@ private JsonNode flattenJobsJson(JsonNode jobJSON) {
             });
             return array;
         } else if (jobJSON.has("job")) {
-            return jobJSON.get("job");
+            return processTargetDatabase(jobJSON.get("job"));
         } else {
             return jobJSON;
         }
     }
 
+    /**
```

Comment:
I don't see any test coverage for this. I think it would be reasonable to do unit tests against this method with a fake jobNode - i.e. no need to run a job first.